src/lib.rs

```diff
@@ -61,7 +61,9 @@ pub struct Response {
     /// An error if there is one, or null
     pub error: Option<error::RpcError>,
     /// Identifier from the request
-    pub id: json::Value
+    pub id: json::Value,
+    /// jsonrpc field, MUST be "2.0"
+    pub jsonrpc: json::Value
```

Comment:
Can you please make this an `Option<json::Value>`? Otherwise there will be parse errors with RPC clients that don't include it.

I know this violates the spec, but right now we're capable of parsing JSONRPC 1.0 and I'd like to keep that.